test/docfx.Test/common/TestHelper.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Microsoft.Docs.Build
+{
+    internal static class TestHelper
+    {
+        public static IEnumerable<(string docsetPath, BuildTestSpec testSpec)> PrepareDocsetsFromSpec(string specPath)
+        {
+            Assert.NotNull(specPath);
+
+            var i = 1;
+            var docsets = new List<(string docsetPath, BuildTestSpec testSpec)>();
+            var specName = specPath.Replace("\\", "/").Replace("specs/", "").Replace(".yml", "");
+            foreach (var testSpec in YamlUtility.DeserializeMany<BuildTestSpec>(File.ReadAllText(specPath)))
+            {
+                var docsetPath = Path.Combine("specs.drop", specName, $"{i++}");
+
+                if (Directory.Exists(docsetPath))
+                {
+                    Directory.Delete(docsetPath, recursive: true);
+                }
+
+                foreach (var (file, content) in testSpec.Inputs)
+                {
+                    var filePath = Path.Combine(docsetPath, file);
+                    Directory.CreateDirectory(Path.GetDirectoryName(filePath));
+                    File.WriteAllText(filePath, content);
+                }
+
+                docsets.Add((docsetPath, testSpec));
+            }
+
+            return docsets;
+        }
+
+        public static void VerifyJsonContainEquals(JToken expected, JToken actual)
+        {
+            if (expected is JArray expectedArray)
+            {
+                Assert.IsType<JArray>(actual);
+                if (actual is JArray actualArray)
```

Comment:
After `Assert.IsType`, `is` should be direct cast `actualArray = (JArray)actual;`